dash/response.py

```diff
@@ -0,0 +1,58 @@
+"""
+Extended Flask Response for use inside callbacks
+"""
+import json
+import plotly
+from flask import Response
+from . import exceptions
+
+
+# pylint: disable=too-many-ancestors
+class DashResponse(Response):
+    """
+    Flask Response extended with option to convert a regular response to
+    valid Dash json-encoded component.
+
+    Return a `DashResponse` object from a Dash callback in order to set
+    other properties of the response like headers or cookies.
+    """
+    def __init__(self, response, **kwargs):
+        self.dash_response = response
```

Comment:
Let's call that output_value instead since it is the return value of a callback and not a response.